pygpudrive/datatypes/roadgraph.py

```diff
@@ -129,37 +129,39 @@ class LocalRoadGraphPoints:
 
     def __init__(self, local_roadgraph_tensor: torch.Tensor):
         """Initializes the global road graph points with a tensor."""
-        self.x = local_roadgraph_tensor[:, :, :, 0]
-        self.y = local_roadgraph_tensor[:, :, :, 1]
-        self.segment_length = local_roadgraph_tensor[:, :, :, 2]
-        self.segment_width = local_roadgraph_tensor[:, :, :, 3]
```

Comment:
Why did you remove a level of indexing? Is this just to reduce the output dim by one?